lib/server.js

```diff
@@ -728,6 +728,7 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
           if (!bitcoreError) {
             txp.inputPaths = _.pluck(txp.inputs, 'path');
             txp.fee = bitcoreTx.getFee();
+            $.checkState(txp.fee < 1e8, 'Fees are too high!');
```

Comment:
So many intertwining libraries with layers... I always forget.

I agree.
1. it should be bitcore-wallet-utils
2. if bitcore is doing a lot of fee checks to begin with, then just a light check.